workshops/book-dash/meeting-notes/2023-09-05-WG.md

```diff
@@ -0,0 +1,57 @@
+    * Working group - how are people selected/or how to do they apply? 
+        * Recommendations for basic criteria
+            * Have attended a Book Dash 
+            * [desirable] Have previously been on the planning committee at least once 
+        * Term length for working group: Two years? Would balance effort on elections/selections, with keeping the WG fresh 
+        * Could revisit options around renewing terms later 
+        * How to engage new people in between rounds? Planning committee/hubs? 
+        * How to manage people who **only** engage as part of the Book Dash? (Q for others as well)
+* Review applications to become part of the planning committee. Arielle has the access
+    * EK & AB would recommend accepting all to join the planning committee for this round - with the expectation that after the Book Dash some will join the Working Group
+    * Logic of this is due to timings an d lack of governanced docs difficult to stand up the WG right now 
+* Start brainstorming any activities that we would like to offer / to start quoting for the Book Dash this round
+    * Discussion around Scriberia images are sometimes duplicated and we struggle to get them into the turing Way itself - should we include them this time or skip and focus on imagathon?
+    * EK: maybe we alternate Book Dashes for Scriberia and go back to a longer time period for the May round
+    * Also recommend we have "imagathons" during the Book Dash this time to help get images into the guides - good for newer team members 
+    * JonnyWritesWords - recommend two sessions due to timezone differences
+    * Social session ideas: learning british sign language, data visualisation, design and accessibility (could we find professional instructors for these?) 
+    * Session in Spanish or different language rooms - possibly with live translation to English?
+      *  Need to speak to localisation team about recommendations for translators and whether Spanish is a good language to start with 
+* Identify or start the conversation about the max number of participants per hub and nr of hubs / rules to open up a hub
+    * Esther not able to join this meeting , would appreciate her input 
+    * Draft Recommendations 
+        * Hub operate on 1-2 days max (have those days coordinated across the Book Dash, to balance online and in-person collaborations) 
+        * In person limits - max 15, recommend 10 to help with coordination, and stress levels!
+        * Haven't yet needed to cover room hire costs but could make provisions for that in the future 
+        * Rules to open up a hub 
+            * Have previously attended a Book Dash 
+            * Have organised an in person event at their institute before (of any kind) 
+            * Adhere to Turing Way CoC 
+            * Provide a summary of in person contributions (light touch, in shared doc)
```

Comment:
Perhaps they should also be able to access some sort of budget? TU Delft hub provided for lunches, snacks and dinner last time which went great. If they do not have access to a budget, what budget is the Turing willing to provide?

Reply:
We could provide the same, snacks, sandwich lunch for the in-person sessions and a dinner.